mu-plugins/blocks/ratings-bars/src/block.json

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "wporg/ratings-bars",
+	"version": "0.1.0",
+	"title": "Ratings (bars)",
+	"category": "design",
+	"icon": "",
+	"description": "The breakdown of ratings displayed as bars for each rating value.",
+	"textdomain": "wporg",
+	"supports": {
+		"html": false
+	},
+	"attributes": {
+		"num_ratings": {
+			"type": "number"
+		},
+		"ratings": {
+			"type": "array",
+			"default": []
+		},
+		"slug": {
+			"type": "string"
+		},
+		"support_url": {
+			"type": "string"
+		}
+	},
```

Comment:
Typically block attributes use camelCase, so this should be `supportUrl` & `numRatings` (maybe `ratingsCount` or `totalRatings` would be a little more understandable? not a strong opinion, I know this reflects the theme object).